### ISSUE TITLE
Copyright added from default config causes COMMENTED_OUT_CODE trigger

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#738]
## Describe the bug
* Run diktat 0.3.0 with default config on file without copyright
* ???
* ~Profit~

## Environment information
* diktat version: 0.3.0

**Comment by aktsay6** [comment]
Do we have a code snippet, where this warning arised?

**Comment by petertrr** [comment]
Yep, copyright text from the default config:
```kotlin
/*
    Copyright (c) Your Company Name Here. 2010-2021
*/

package com.example
```

Error was observed when diktat adds this text, but I think it shouldn't matter.

**Comment by petertrr** [comment]
Copyright in default config (`diktat-rules/resources/diktat-analysis`), as stated in #737 , can be disabled in this issue as well.
